bin/configs/spring-boot-3.yaml

```diff
@@ -0,0 +1,12 @@
+generatorName: spring
+outputDir: samples/openapi3/server/petstore/springboot-3
+inputSpec: modules/openapi-generator/src/test/resources/3_0/petstore.yaml
+templateDir: modules/openapi-generator/src/main/resources/JavaSpring
+additionalProperties:
+  groupId: org.openapitools.openapi3
+  documentationProvider: springdoc
```

Comment:
probably this should be defaulted to `springdoc` without ability to override it. same for `useJakartaEe=true`

Reply:
**springFoxDocumentationProvider** and swagger1AnnotationLibrary are not accepted with **useSpringBoot3**.